bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.html

```diff
@@ -9,26 +9,32 @@ <h2>Create Secret</h2>
       <form [formGroup]="createSecretForm">
         <div class="mb-3" fxLayout="row" fxLayoutAlign="none end" fxLayoutGap="15px">
           <dt-form-field>
-            <dt-label>Name *</dt-label>
-            <input formControlName="name" type="text" required dtInput placeholder="Secret name" />
+            <dt-label class="required">Name</dt-label>
+            <input formControlName="name" type="text" required dtInput placeholder="Secret name" uitestid="keptn-secret-name-input" />
             <dt-error *ngIf="getFormControl('name')?.errors?.required">Must not be empty</dt-error>
             <dt-error *ngIf="getFormControl('name')?.errors?.pattern">Name must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character</dt-error>
           </dt-form-field>
+          <dt-form-field>
+            <dt-label class="required">Scope</dt-label>
+            <dt-select formControlName="scope" class="mr-2 item" placeholder="Choose scope" aria-label="Choose your task" uitestid="keptn-secret-scope-input">
```

Comment:
Could you set a fixed width for it? Depending on which item is selected, the width of the control changes.